### PR TITLE
Made tests for the project

### PR DESCRIPTION
**Pull request by mnaibei**
- Used jest and react testing library to make tests for math-magicians app
- Checked linters
- Created PR

### CODE REVIEW

**Comment by DammyShittu** [comment]
Hi @mnaibei,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Ensure you collaborate with your coding partner.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/test/calcLogic.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  it('should return a sum if "+" is assigned between two numbers', () => {
+    const result = operate(1, 5, '+');
+    expect(result).toEqual('6');
+  });
+
+  it('should return a multiplication if "x" is assigned between two numbers', () => {
+    const result = operate(10, 3, 'x');
+    expect(result).toEqual('30');
+  });
+});
```
Comment (danmainah) [inline src/test/calcLogic.test.js:25]: - [ ] @mnaibei and @RHK-MICROVERSE great work on your tests, however, you didn't cover all functionalities and scenarios while testing the `operate.js` and `calculate.js` modules.

For instance, in the calculate.js file the if statement shown below represents a scenario, the same case applies to the operate.js file
image

- [ ] @team While you did a great job testing the components you forgot to add tests for Navbar and Display components, kindly consider adding tests for these components as it was part of the project requirement as shwon below

image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @mnaibei and @RHK-MICROVERSE , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ All tests are passing 
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] @team great work testing your application, while running the app I discovered your quotes are not loading , it would be nice if you fixed the error causing this problem
image


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/test/calcLogic.test.js (hunk 1 above)
Comment (mnaibei) [reply src/test/calcLogic.test.js:25]: @danmainah the quotes are not working because I have hidden the API key from github

**Review by uwadonat — APPROVED** [review]
## Hi @mnaibei,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Good project structure ✔️
- Professional readme file ✔️
- No linter errors ✔️
- Amazing design ✔️
thanks-awesome

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**